packages/common-ts/src/base-service/base-service-v2.ts

```diff
@@ -330,6 +331,7 @@ export abstract class BaseServiceV2<
           verify: (req, res, buf, encoding) => {
             ;(req as any).rawBody = buf?.toString(encoding || 'utf8') || ''
           },
+          ...this.params.bodyParserParams,
```

Comment:
Will this cause a problem if the params are undefined? It looks like they are optional

Reply:
It won't because ...undefined and ...(undefined ?? {} or ...{} or ...null are all the same thing in javascript.  That said being explicit is a good thing
```suggestion
          ...(this.params.bodyParserParams ?? {}),
```